variables.tf

```diff
@@ -120,6 +120,12 @@ variable "cluster_performance_insights_retention_period" {
   default     = null
 }
 
+variable "cluster_monitoring_interval" {
+  description = "Interval, in seconds, in seconds, between points when Enhanced Monitoring metrics are collected for the DB cluster. To turn off collecting Enhanced Monitoring metrics, specify 0. The default is 0. Valid Values: 0, 1, 5, 10, 15, 30, 60"
+  type        = number
+  default     = 0
```

Comment:
```suggestion
  default     = null
```

Reply:
This is similar to what we're doing for `var.monitoring_interval`: https://github.com/terraform-aws-modules/terraform-aws-rds-aurora/blob/685b01b64754953db29a89f79488f560e56e5731/variables.tf#L415-L419
That way we can use `var.cluster_monitoring_interval > 0` in places to determine if we should configure a monitoring role since it must be greater than `0` in that case. Or should we let `var.create_monitoring_role` handle that on its own? Let me know what you think.

Reply:
Good point, I agree.